include/ginkgo/core/log/convergence.hpp

```diff
@@ -94,6 +102,18 @@ class Convergence : public Logger {
             new Convergence(exec, enabled_events));
     }
 
+    /**
+     * Returns true if the solver has stopped.
```

Comment:
nit: this seems to be left-over from before
```suggestion
     * Returns true if the solver has converged.
```